pkg/R/RDD.R

```diff
@@ -839,6 +727,7 @@ setMethod("take",
             resList
           })
 
+setClassUnion("missingOrInteger", c("missing", "integer"))
```

Comment:
Could we also move this to the generics.R file ? I think we use missingOrInteger and missingOrNumeric in a bunch of places, so it'll be good to unify them